test/compiler/scale.test.ts

```diff
@@ -2,7 +2,6 @@
 
 import {expect} from 'chai';
 
-var d3 = require('d3');
```

Comment:
Yes, I just removed it because it wasn't doing anything. It just surprised me that it was there.